volpiano_display_utilities/latin_word_syllabification.py

```diff
@@ -229,33 +240,45 @@ def _get_syl_bound_position(ltrs_btw_vow_grps: str) -> Tuple[int, str]:
     num_ltrs_btw_vow_grps = len(ltrs_btw_vow_grps)
     # Default case: syllable boundary immediately follows previous
     # vowel group.
-    syl_bound = 0
     if num_ltrs_btw_vow_grps == 0:
-        split_case = "Hiatus"
-    elif ltrs_btw_vow_grps[0] == "x":
+        return 0, "Hiatus"
+    if ltrs_btw_vow_grps[0] == "x":
+        return 1, "X is double consonant"
+    if num_ltrs_btw_vow_grps == 1:
+        return 0, "1 consonant between vowels"
+    # If the first letter of the consonant sequence is a nasalized consonant,
+    # we add it to the prior syllable and treat the remaining consonants
+    # as if they were the only consonants between the vowel groups.
+    if ltrs_btw_vow_grps[0] in _NASALIZED_CONSONANTS:
         syl_bound = 1
-        split_case = "X is double consonant"
-    elif num_ltrs_btw_vow_grps == 1:
-        split_case = "1 consonant between vowels"
-    elif num_ltrs_btw_vow_grps == 2:
-        if ltrs_btw_vow_grps not in _CONSONANT_GROUPS or ltrs_btw_vow_grps == "nc":
-            syl_bound = 1
-            split_case = "2 consonants between vowels"
-        else:
-            split_case = "2 consonants between vowels (consonant group)"
+        ltrs_btw_vow_grps = ltrs_btw_vow_grps[1:]
+        num_ltrs_btw_vow_grps -= 1
+        split_case_nasal_cons_tag = " (first consonant nasaslized)"
+        num_consonants = num_ltrs_btw_vow_grps + 1
```

Comment:
it seems that `num_consonants` is always equal to `num_ltrs_btw_vow_grps` before the latter gets `-= 1`ed. could we just set this outside the `if` statement?